website/source/docs/job-specification/periodic.html.md

```diff
@@ -39,6 +39,8 @@ consistent evaluation when Nomad spans multiple time zones.
 
  - The job's [scheduler type][batch-type] must be `batch`.
 
+~> To resubmit the non-periodic job as a periodic one you should run **nomad stop -purge «job name»** first. This is expected behavior and is to ensure that this change has been intentionally made by an operator.
```

Comment:
I think the call out is slightly too aggressive.

I would maybe suggest rewording such as:
```
- A job can not be updated to be periodically. Thus, to transition an existing job to be periodic, you must first run...
```